www/js/cameraAPI.js

```diff
@@ -0,0 +1,38 @@
+function capturePhoto(){
+    navigator.camera.getPicture(uploadPhoto,null,{sourceType:1,quality:60});
+}
+
+function uploadPhoto(data){
+// this is where you would send the image file to server
+
+	cameraPic.src = "data:image/jpeg;base64," + data;
+	// Successful upload to the server
+	navigator.notification.alert(
+		'Your Photo has been uploaded',  // message
+		okay,                           // callback
+	    'Photo Uploaded',              // title
+	    'OK'                          // buttonName
+	);
+
```

Comment:
please fix the styling of these lines so they are spaced and indented consistently.